### PR TITLE
Add a troubleshooting topic for missing FBSnapshotTestCase images.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.clubhouse.io/buddybuild/story/6902/add-troubleshooting-entry-for-users-not-seeing-fbsnapshottestcase-images-in-test-results

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://134-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://134-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://134-bb-docs.surge.sh

File: troubleshooting/ios/fbsnapshottestcase.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+= FBSnapshotTestCase images missing from test results
+
+When the `OS_ACTIVITY_MODE` environment variable is set to `disabled`,
+this removes all system logs. It also removes `NSLog` from your
+application on iOS 11.
+
+The solution is to set `OS_ACTIVITY_MODE=default`, which continues to
```
Comment (nathanshox) [inline troubleshooting/ios/fbsnapshottestcase.adoc]: Does this sentence read a little cumbersome?

Wondering if something like 
> ..., but keeps `NSLog` with log lines containing the paths to the generated screenshots.

**Comment by bbdocs** [comment]
See the website here: http://134-bb-docs.surge.sh

**Comment by nathanshox** [comment]
Lovely stuff!

**Comment by bbewan** [comment]
@nathanshox You need to accept your invitation to buddybuild-public before your approval would let me merge this PR. :-)

**Pull request merged**